### PR TITLE
Remove sudo from pacscripts

### PR DESCRIPTION
**Pull request by Korkmatik**

### CODE REVIEW

File: packages/android-studio/android-studio.pacscript [hunk 1]
```diff
@@ -33,14 +33,14 @@ postinst(){
...
 }
 removescript(){
     # Removing .desktop file
-    sudo rm /usr/share/applications/android-studio.desktop
-    sudo update-desktop-database -q
+    rm /usr/share/applications/android-studio.desktop
+    update-desktop-database -q
     # Removing launcher
-    sudo rm /usr/bin/android-studio
+    rm /usr/bin/android-studio
```
Comment (Henryws) [inline packages/android-studio/android-studio.pacscript:45]: This should still be sudo as it's not trying to install to `/usr/src/pacstall` which is owned by the user

File: packages/android-studio/android-studio.pacscript [hunk 2]
```diff
@@ -21,7 +21,7 @@ install() {
     cp -r license "$STOWDIR/$name/opt/android-studio/"
     cp -r plugins "$STOWDIR/$name/opt/android-studio/"
     # Creating a launcher in $PATH
-    sudo ln -s "$STOWDIR/$name/opt/android-studio/bin/studio.sh" "/usr/bin/android-studio"
+    ln -s "$STOWDIR/$name/opt/android-studio/bin/studio.sh" "/usr/bin/android-studio"
```
Comment (NarutoXY) [inline packages/android-studio/android-studio.pacscript:24]: This one should probably require `sudo` as we are symlinking to a file owned by `root`

File: packages/android-studio/android-studio.pacscript [hunk 3]
```diff
@@ -33,14 +33,14 @@ postinst(){
     echo 'Icon=/opt/android-studio/bin/studio.png'          >> /usr/share/applications/android-studio.desktop
     echo 'Exec=/usr/bin/android-studio'                     >> /usr/share/applications/android-studio.desktop
     echo 'Name=Android Studio'                              >> /usr/share/applications/android-studio.desktop
-    sudo update-desktop-database -q
+    update-desktop-database -q
```
Comment (wizard-28) [inline packages/android-studio/android-studio.pacscript:36]:
```suggestion
    sudo update-desktop-database -q
```

I think this needs `sudo`

**Comment by wizard-28** [comment]
Well... Closing this as fakeroot is a thing of the past now.

**Pull request closed without merging**